models/Daily_naive_darts/requirements.txt

```diff
@@ -0,0 +1,2 @@
+enfobench>=0.3.3
+darts>=0.27.0
```

Comment:
please use the u8darts package as it is much smaller than the darts package (it does not have all the pytorch dependencies included that we do not need for this example)

Reply:
I have updated the package to u8darts